src/compiler/binder.ts

```diff
@@ -1835,6 +1833,8 @@ namespace ts {
                 case SyntaxKind.ClassStaticBlockDeclaration:
                     return ContainerFlags.IsContainer | ContainerFlags.IsControlFlowContainer | ContainerFlags.HasLocals | ContainerFlags.IsFunctionLike;
 
+                case SyntaxKind.GetAccessor:
+                case SyntaxKind.SetAccessor:

```

Comment:
This move doesn't seem right -- `get/set` aren't function expressions. Do the tests still pass with this change reverted?

Reply:
reverting just this change causes the new test to fail. The goal here was (admittedly opaquely) to have `bindContainer` bind the getter/setter node to the fresh start flow, which returning these container flags succeeded at doing (the desired flow binding occurs if the `containerFlags` for a given container are either `ContainerFlags.IsFunctionExpression` or `ContainerFlags.IsObjectLiteralOrClassExpressionMethod`). Your comments about consolidating `isObjectLiteralOrClassExpressionMethod` makes it look like moving these switch cases to be with the `MethodDeclaration` case makes sense.

This made me realize I should have made explicit the intent to attach `GetAccessorDeclaration` and `SetAccessorDeclaration`s to `FlowStart.node`; doing so has the possibly unfortunate side effect of changing the shape of the `FlowStart` interface in 2 publicly exposed interface files. Not sure what the policy is on that.